README.md

```diff
@@ -8,31 +10,25 @@ Exposing DanielSWolf / rhubarb-lip-sync as a service
 docker build -t korerorero-mouth-shapes:latest .
 ```
 
-To run
+__To run__
 
 ```bash
 docker run -p 3000:3000 korerorero-mouth-shapes:latest
 ```
 
-To run tests on local code changes
+__To run tests on local code changes__
 
 ```bash
 docker run -it  -v $PWD/src:/usr/src/app/src korerorero-mouth-shapes:latest /usr/bin/npm test
```

Comment:
When running this command I get this error: 
![image](https://user-images.githubusercontent.com/13788498/80174643-5aa84a80-8647-11ea-866d-d01326a71a5a.png)

Reply:
Works now:
![image](https://user-images.githubusercontent.com/13788498/80176367-7530f280-864c-11ea-93b0-b38fa6b0fc94.png)